pandas/conftest.py

```diff
@@ -521,6 +521,11 @@ def index_or_series_obj(request):
 # ----------------------------------------------------------------
 # DataFrames
 # ----------------------------------------------------------------
+@pytest.fixture
+def empty_frame():
```

Comment:
This is scattered across a few modules right now that I didn't touch here; can use further here or in a follow up

Reply:
followup is fine.